source/conf.py

```diff
@@ -383,3 +396,8 @@
 # -- Options for mathjax --------------------------------------------------
 mathjax_path = ('https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/'
                 'MathJax.js?config=TeX-AMS-MML_HTMLorMML')
+
+
+def setup(app):
+    app.connect('build-finished', cleanup_plugin_rst)
+    return {'version': '0.0.1'}
```

Comment:
Please move this up into the new "dynamic content" section. Also, I think you can drop the `return` statement, since this isn't really a new extension, we don't need to declare a version identifier.